component/badge/badge.css

```diff
@@ -27,11 +27,43 @@
   background-color: var(--default-badge-color);
 }
 
+.badge-text {
+  width: max-content;
+  height: fit-content;
+  padding: 4px 6px;
```

Comment:
You can use rem instead of px!
other than that looks nice
you can merge it🚀

Reply:
done